### PR TITLE
Avoids logging results when they are null

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

Fixes: 
```2016-10-26 22:01:47 DEBUG    SEARCHQUEUE-MANUAL-282401 :: [OMGWTFNZBs] :: [683951e] Found result: None```

### CODE REVIEW

File: medusa/providers/nzb/omgwtfnzbs.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def search(self, search_strings, age=0, ep_obj=None):
 
                 for item in response:
                     try:
-                        if not self._get_title_and_url(item):
+                        if not all(self._get_title_and_url(item)):
```
Comment (fernandog) [inline medusa/providers/nzb/omgwtfnzbs.py]: @medariox need to do this to all providers as the old check return true even if one of items in tuple are none

Comment (medariox) [inline medusa/providers/nzb/omgwtfnzbs.py]: If this really needs to be done to all providers (can't check right now, not even sure what the method exactly does), it should be part of the method.

Comment (fernandog) [inline medusa/providers/nzb/omgwtfnzbs.py]: that method returns title and url from the item. Its in generic provider

**Comment by duramato** [comment]
```
2016-10-27 23:05:48 DEBUG    SEARCHQUEUE-MANUAL-282401 :: [OMGWTFNZBs] :: [c7cab9a] Found result: None
2016-10-27 23:05:48 DEBUG    SEARCHQUEUE-MANUAL-282401 :: [OMGWTFNZBs] :: [c7cab9a] Found result: None
2016-10-27 23:05:48 DEBUG    SEARCHQUEUE-MANUAL-282401 :: [OMGWTFNZBs] :: [c7cab9a] Found result: None
```

**Pull request merged**